model/clusters_mgmt/v1/alert_info_type.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Provides information about a single alert firing on the cluster.
+struct AlertInfo {
+	// The alert name.  Multiple alerts with same name are possible (and the API does not currently expose details to distinguish them).
+	Name     string
+
+	// The alert severity.
+        Severity AlertSeverity
```

Comment:
There seems to be a mix of spaces and tabs indenting this file. Make sure to use only tabs.